vMenuServer/config/languages.json

```diff
@@ -0,0 +1,1631 @@
+{
+	"american": {},
```

Comment:
Also please split these languages up in different files, that way maintaining custom language versions is much easier for people when a new language is added or updated.

Reply:
I agree. When I first tried to support multiple languages, it was split into multiple files.